src/main/java/com/microsoft/jenkins/appservice/commands/GitDeployCommand.java

```diff
@@ -56,6 +55,7 @@
     private static final String DEPLOY_COMMIT_MESSAGE = "Deploy ${BUILD_TAG}";
     private static final String DEPLOY_BRANCH = "master";
     private static final String DEPLOY_REMOTE_BRANCH = "origin/" + DEPLOY_BRANCH;
+    private static final String GIT_ADD_ALL_PARAMETER = ".";
```

Comment:
The interface here need a filePattern instead of a argument. When I pass "-A", the code cannot work properly and test cases will fail. "-A" and "." are equivalent in Git V2.*, so maybe we should check the git version here and use different strategy.

Reply:
`.` is better for jgit since it is hard code in their codes. Use `.` will have better performance too.